test/Microsoft.TestPlatform.TestUtilities/IntegrationTestEnvironment.cs

```diff
@@ -42,8 +42,8 @@ public IntegrationTestEnvironment()
             {
                 // Running in VS/IDE. Use artifacts directory as root.
                 this.runningInCli = false;
-                this.testPlatformRootDirectory = Path.GetFullPath(@"..\..\..");
-                this.TestAssetsPath = Path.Combine(this.testPlatformRootDirectory, @"artifacts\test\TestAssets");
+                this.testPlatformRootDirectory = Path.GetFullPath(@"..\..\..\..\..");
```

Comment:
Tests current directory is test assembly directory. finding path from there, added comment.